### ISSUE TITLE
scale parameter value range  [0,2], why?

### ISSUE THREAD

**Issue by oliverxudd** [seungryong/LAF#1]
Hi, seungryong, much thanks for the nice work and really neat code!
I just want to ask a question, why did you set scale parameter at the range of [0, 2], as in your models.py at line 153
`scale = 2*self.sigmoid(self.scale_bn3(self.scale_conv3(x)))`
I mean, the idea of using different receptive field for each pixel is very cool, but why set the max value at 2? isn't it too small?

**Comment by keepuplearning** [comment]
> 

I also have this question! And what's more, I have another question, that is why he use the step_y like "grid_enlarge[:,0::3,0::3,:] = grid+torch.cat(((-1)*step_y*scale_t,(-1)*scale_t),3)" .
